.github/workflows/ci.yml

```diff
@@ -305,6 +305,16 @@ jobs:
     if: needs.detect-noop.outputs.noop != 'true'
 
     steps:
+      - name: Cleanup Disk
+        uses: jlumbroso/free-disk-space@main
+        with:
+          android: true
+          dotnet: true
+          haskell: true
```

Comment:
the names of these values don't look applicable to our project, and the [readme](https://github.com/jlumbroso/free-disk-space#options) just says

> Most of the options are self-explanatory.

wondering if these are actually needed, but i do see its the same values we already put into #4391 and #4392, so i'm not inclined to rock the boat now 😂